.idea/runConfigurations/Flutter_Plugin__tests_.xml

```diff
@@ -0,0 +1,29 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Curious about the double underscore in the filename.  Is this a convention?  Cool either way! 👍